src/components/Whiteboard/ElementBehaviors/Resizable/utils.ts

```diff
@@ -80,6 +81,89 @@ export function calculateDragDimension(
   };
 }
 
+export function calculateLineDragDimensions(
+  handlePosition: ResizeHandlePosition,
+  dimensions: Dimensions,
+  dragX: number,
+  dragY: number,
+): Dimensions {
+  if (
+    dimensions.elementKind !== 'line' ||
+    handlePosition === 'top' ||
+    handlePosition === 'right' ||
+    handlePosition === 'bottom' ||
+    handlePosition === 'left'
+  ) {
+    return dimensions;
+  }
+
+  const start = dimensions.points[0];
```

Comment:
We should enable strict boolean checks. This may lead to start being undefined and weird things to happen.

If we consider a path as invalid if having less than two points, what about declaring the points in in the dimension type like this?

```
points: [Point, Point, ...Point[]];
```

Reply:
I had the same idea and decided to stick to what we already have for lines.